LoopKitUI/Extensions/Environment+Colors.swift

```diff
@@ -110,7 +110,8 @@ private struct ChartColorPaletteKey: EnvironmentKey {
                                                                    axisLabel: .secondaryLabel,
```

Comment:
These colors appear to be fallbacks when there is no value specified in the swift environment. systemGreen seems right as that is the same fallback used in Loop/LoopUI/Extensions/UIColor.swift